piksi_tools/pickle_to_json.py

```diff
@@ -0,0 +1,52 @@
+from sbp_log_analysis.death import ScreamingDeathTest
+from sbp_log_analysis.heartbeat import HeartbeatTest
+from sbp_log_analysis.junit import metrics_xml_to_dict
+from sbp_log_analysis.runner import SBPTestRunner
+from sbp_log_analysis.solution import SolutionTest
+from sbp_log_analysis.startup import StartupTest
+from sbp_log_analysis.test import SBPTestSuite
+from sbp_log_analysis.thread import ThreadTest
+from sbp_log_analysis.uart import UARTTest
+from sbp_log_analysis.utils import deep_eq
+from sbp_log_analysis.utils import deep_eq
+from sbp.client.loggers.json_logger import JSONLogIterator
+from sbp_log_analysis.tracking import TrackingTest
+from sbp.client.loggers.pickle_logger import PickleLogIterator
+import json
+import struct
+import sbp
+import sbp.table
+
+def rewrite(files):
+  """ Rewrite pickle file as a json file with a similar name
+  """
+  for f in files:
+    with open(f, "r") as handle:
+      with open(f + ".json", 'w+') as log_file:
+        with PickleLogIterator(handle) as log:
+          for delta, timestamp, msg in log.next():
+            if msg.msg_type == 26:
```

Comment:
Did you want to leave this mapping in? This is specfically for handling the old ephemeris message type cases.